src/components/WorkOrder/WorkOrderDetails.test.js

```diff
@@ -102,6 +119,24 @@ describe('WorkOrderDetails component', () => {
       )
       expect(asFragment()).toMatchSnapshot()
     })
+    it('should render migrated work order references correctly', () => {
```

Comment:
maybe we could add a comment here so it is clear how is "correctly": 
// if work order is 648707 then it should render is as 0064 8707 

something like that

Reply:
Added